libbeat/docs/monitoring/monitoring-beats.asciidoc

```diff
@@ -46,7 +46,8 @@ xpack.monitoring:
   elasticsearch:
 --------------------
 
-Otherwise, you must specify additional configuration options. For example:
+If you are using a different output then Elasticsearch, you must specify
```

Comment:
I thinks these details are still a little hard to follow. Are you saying that users *must* send monitoring data to the production cluster? If so, you should also modify the sentence that begins on line 37 (remove "and you want to use the same {es} production cluster and credentials"). So I would change this to say something like:

```
. Add the `xpack.monitoring` settings in the {beatname_uc} configuration file. If you configured {es} output,
specify the following minimal configuration:
.
.
.
If you configured a different output, such as {ls}, you must specify additional configuration options.
For example:
.
.
.
NOTE: Currently you must send monitoring data to the same cluster as all other events. If you configured the
{es} output, do not specify additional hosts in the monitoring configuration. 
```

Reply:
Thanks for the proposal. Adjusted accordingly.